tests/integration/test_scan.py

```diff
@@ -131,6 +131,41 @@ def _run_tests(op, n0, shape, dt, axis, out0, outtype):
 ]
 
 
+@pytest.mark.parametrize(
+    "dtype, outtype",
+    [
+        (np.int16, np.float64),
+        (np.complex64, np.float64),
+        (np.float32, np.int64),
+    ],
+    ids=str,
+)
+@pytest.mark.parametrize(
+    "op",
+    [
+        pytest.param("cumsum", marks=pytest.mark.xfail),
```

Comment:
What exactly is the failure for `cumsum` specifically?

Reply:
Moved the explanation to a more relatable location.